tasks/rabbitmq.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: rabbitmq
+  namespace: tasks
+spec:
+  revisionHistoryLimit: 0
+  replicas: 2
+  selector:
+    matchLabels:
+      app: rabbitmq
+  serviceName: rabbitmq
+  template:
+    metadata:
+      labels:
+        app: rabbitmq
+    spec:
+      imagePullSecrets:
+        - name: registry.internal.aleemhaji.com
+      containers:
+        - name: rabbitmq
+          image: registry.internal.aleemhaji.com/rabbitmq:3.11.15-management
+          ports:
+            - containerPort: 5672
+            - containerPort: 15672
+          command:
+            - sh
+            - -eufc
+            - |
+              printf 'cluster_formation.classic_config.nodes.1 = %s\ncluster_formation.classic_config.nodes.2 = %s\n' \
+                rabbit@rabbitmq-0.rabbitmq rabbit@rabbitmq-1.rabbitmq > /var/lib/rabbitmq/rabbitmq.conf
+              rabbitmq-plugins --offline enable rabbitmq_shovel rabbitmq_shovel_management
+              RABBITMQ_NODENAME=rabbit@$HOSTNAME.rabbitmq rabbitmq-server --erlang-cookie "${ERLANG_COOKIE}"
+          env:
+            - name: RABBITMQ_DEFAULT_USER
+              valueFrom:
+                secretKeyRef:
+                  name: rabbitmq-secrets
+                  key: username
+            - name: RABBITMQ_DEFAULT_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: rabbitmq-secrets
+                  key: password
+            - name: ERLANG_COOKIE
+              valueFrom:
+                secretKeyRef:
+                  name: rabbitmq-secrets
+                  key: .erlang-cookie
+            - name: RABBITMQ_USE_LONGNAME
+              value: "true"
+            - name: RABBITMQ_CONFIG_FILE
+              value: /var/lib/rabbitmq/rabbitmq.conf
+          volumeMounts:
+            - name: rabbitmq-persistent-storage
+              mountPath: /var/lib/rabbitmq
+      volumes:
+        - name: rabbitmq-erlang-cookie
+          secret:
+            secretName: rabbitmq-secrets
+        - name: rabbitmq-persistent-storage
+          emptyDir: {}
```

Comment:
Need to figure this out before merging. Need to have a proper volume management set up, so there isn't constant data loss.